cli/src/main/java/io/aklivity/zillabase/cli/internal/commands/start/ZillabaseStartCommand.java

```diff
@@ -1535,6 +1535,7 @@ private String generateHttpAsyncApiSpecs(
                 }
 
                 String label = matcher.reset(name).replaceAll(match -> match.group(2).toUpperCase());
+                name = matcher.reset(name).replaceAll(match -> match.group(2));
```

Comment:
This doesn't look right to me.
Is the only difference between `label` and updated `name` is upper case?